### PR TITLE
Feature/49170 add liquibase files to create database table

### PR DESCRIPTION
**Pull request by Niek-Ritense**

### CODE REVIEW

File: zgw/search/src/main/resources/liquibase/changelog/20230124-add-search-definition.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <property name="booleanFalseValue" value="0" dbms="mysql"/>
+    <property name="booleanFalseValue" value="false" dbms="postgresql"/>
+
+    <changeSet id="add-search_definition" author="Ritense">
+        <createTable tableName="search_definition">
```
Comment (Klaas-Ritense) [inline zgw/search/src/main/resources/liquibase/changelog/20230124-add-search-definition.xml:26]: This table isn't needed. The story s only about search-list-columns

**Review by Klaas-Ritense — APPROVED** [review]

File: docker-resources/docker-compose-base-test.yml [hunk 2]
```diff
@@ -46,3 +46,4 @@ services:
 #   url: jdbc:postgresql://localhost:3336/plugin-valtimo-test
 #   url: jdbc:postgresql://localhost:3337/verzoek-test
 #   url: jdbc:postgresql://localhost:3338/portaaltaak-test
+#   url: jdbc:postgresql://localhost:3339/search-test
```
Comment (Klaas-Ritense) [inline docker-resources/docker-compose-base-test.yml:49]: It looks like port 3339 has been taken

Comment (Niek-Ritense) [reply docker-resources/docker-compose-base-test.yml:49]: fixed in this pr: https://github.com/valtimo-platform/valtimo-backend-libraries/pull/504

**Comment by ivo-ritense** [comment]
It looks like this PR is incomplete and Klaas told me he looked at a different PR in which this code is also included. Can we close this PR?

**Pull request closed without merging**